docs-new/docs/graphql/core/api-reference/graphql-api/query.mdx

```diff
@@ -107,11 +101,11 @@ subscription {
 
 <div class="parsed-literal">
 
-[SimpleObject](#SimpleObject) \| [AggregateObject](#AggregateObject)
+[SimpleObject](#simple-object) \| [AggregateObject](#aggregate-object)
```

Comment:
no hyphen in both params (`simpleobject`) & (`aggregateobject`)

please fix at respective headers too.

Reply:
fixed.